mlos_bench/mlos_bench/services/config_persistence.py

```diff
@@ -401,11 +424,13 @@ def load_environment(self, json_file_name: str,
         """
         config = self.load_config(json_file_name, ConfigSchema.ENVIRONMENT)
         assert isinstance(config, dict)
-        return self.build_environment(config, tunables, global_config, service)
+        return self.build_environment(config, tunables, global_config, parent_args, service)
 
-    def load_environment_list(self, json_file_name: str,
+    def load_environment_list(self,     # pylint: disable=too-many-arguments
```

Comment:
Could get rid of that pylint warning by changing these all to `*` to require named params, but that change was more extensive and verbose so I skipped it for now.  Happy to change it in a future PR if we want to.

Reply:
sounds good! I'll fix it later when I have cycles